news/src/main/java/org/exoplatform/platform/qa/ui/news/pageobject/NewsComposerManagement.java

```diff
@@ -223,5 +221,18 @@ public void addUserPublisher(String publisher, String username) {
     ELEMENT_MEMBERSHIP_SELECT.selectOptionByValue(publisher);
     NewsLocator.ELEMENT_SAVE_BUTTON.click();
   }
+public void PIN_UNPIN_NEWS_DETAILS ()
+{
+    NEWS_DETAILS_BUTTON_PIN_UNPIN.click();
+
+}
+public void ConfirmPinUnPinArticle ()
+{
+    CONFIRM_PIN_ARTICLE.click();
+}
+public void PIN_NEWS_FROM_CREATIONFORM()
+{  NewsCreationFrom_PINUNPIN_BUTTON.click();}
 
+public void UNPIN_NEWS_FROM_CREATIONFORM()
+  {  NewsCreationFrom_PINUNPIN_BUTTON.click();}
```

Comment:
Take care of the indentation. Use eXo formatting : http://developer.exoplatform.org/#id-formatter-settings